wherescape/connectors/hubspot/python_hubspot_check_missing_columns.py

```diff
@@ -92,6 +95,11 @@ def hubspot_check_missing_columns(
 
 
 def compare_load_and_ds(wherescape_instance, table_rows):
+    """
+    Function that compares the ds table rows with the supplied table_rows. It
+    will push the diffence to the load table to be load table which will be
```

Comment:
typo and unclarity: suggested line: `will push the difference to the load table`